include/sway/output.h

```diff
@@ -84,6 +84,8 @@ void output_damage_whole_container(struct sway_output *output,
 
 struct sway_output *output_by_name_or_id(const char *name_or_id);
 
+struct sway_output *all_output_by_name_or_id(const char *name_or_id);
```

Comment:
Can you add comments to this and to `output_by_name_or_id` clarifying the difference between them?